tests/unittests/Makefile

```diff
@@ -207,7 +207,7 @@ MSP430_BOARDS := chronos \
                  wsn430-v1_4 \
                  z1 \
                  #
-DISABLE_TEST_FOR_MSP430 := tests-relic tests-spiffs tests-cpp_%
+DISABLE_TEST_FOR_MSP430 := tests-relic tests-cpp_%
```

Comment:
This blacklisting needs to also be moved to `tests/pkg_spiffs`.

Reply:
Should be okay now